lib/editor/result.js

```diff
@@ -0,0 +1,281 @@
+'use babel'
+import {CompositeDisposable, Emitter} from 'atom'
+import ResultView from './result-view'
+import fastdom from 'fastdom'
+
+let layers = {}
+
+export default class Result {
+  constructor (editor, lineRange, opts) {
+    this.emitter = new Emitter()
+    this.disposables = new CompositeDisposable()
+
+    this.editor = editor
```

Comment:
Aside from the GC, though, we also have things like state in Julia that needs to be explicitly disposed of. I don't remember exactly how that works so this may not be the issue.